src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfaceProgress.cs

```diff
@@ -166,6 +213,10 @@ class ConsoleHostUserInterface : System.Management.Automation.Host.PSHostUserInt
 
         private ProgressPane _progPane = null;
         private PendingProgress _pendingProgress = null;
+        // The timer set up 'progPaneUpdateFlag' every 'UpdateTimerThreshold' milliseconds to update 'ProgressPane'
+        private Timer _progPaneUpdateTimer;
+        private const int UpdateTimerThreshold = 200;
+        private int progPaneUpdateFlag;
```

Comment:
I know the default values for `_progPaneUpdateTimer` and `progPaneUpdateFlag` will be null and 0, but I recommend to explicitly initialize the fields with a default value. It's also for keeping consistency as the rest fields of the class.